source/docs/software/hardware-apis/index.rst

```diff
@@ -44,6 +42,21 @@ Motor controllers come in lots of shapes, sizes and feature sets. This is the fu
 - Victor SPX Motor Controller (P/N: 217-9191, 17-868388, am-3748)
 - Venom Motor with Controller (P/N BDC-10001) for controlling integral motor only​
 
+Sensors
+-------
+
+Sensors are incredibly useful for gathering information about the robot and it's environment. Some of the most popular sensors include:
+
+- Accelerometers
+- Gyroscopes
+- Ultrasonics
+- Counters
+- Encoders
+- Analog Inputs
```

Comment:
Analog Input / Digital Input aren't sensors.  They are a way to read a sensor.

Reply:
Counters aren't either